test/client-old/spec/controllers/settingsCtrlSpec.js

```diff
@@ -334,4 +336,42 @@ describe('Settings Controller', function () {
       });
     });
   });
+
+  context('Fixing character values', function () {
+    describe('#restore', function () {
+      it('doesn\'t update character values when level is less than 1', function () {
+        scope.restoreValues = {
+          stats: {
+            hp: 0,
+            exp: 0,
+            gp: 0,
+            lvl: 0,
+            mp: 0,
+          },
+          achievements: {
+            streak: 0,
+          },
+        };
+        scope.restore();
```

Comment:
I'm not sure this is correct. You have stubbed restore above, so the function won't be called, thus, you expectation below will never be true leading to this test not doing anything.